demovideo/index.html

```diff
@@ -0,0 +1,29 @@
+<html>
+
+<head>
+    <title>connect-rtc-js demo video</title>
```

Comment:
updated. video demo page was removed in the new iteration, audio & video logic are now merged into the same demo page.